.travis.yml

```diff
@@ -1,10 +1,15 @@
 language: php
 sudo: false
 
+## Cache composer bits
+cache:
+  directories:
+    - $HOME/.composer/cache
 php:
-  - 5.5
```

Comment:
So if they dropped support for <= 5.6 shouldn't you remove 5.6 as well?  Or did you mean < 5.6?